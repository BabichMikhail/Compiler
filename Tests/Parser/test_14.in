a + (b + c) * 100000