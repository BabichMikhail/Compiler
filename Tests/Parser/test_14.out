      100000
   *
         c
      +
         b
+
   a
